src/components/custom/_dhero-dmt5.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import { localize } from 'components/localization'
 import { Flex } from 'components/containers'
 import { QueryImage, Header } from 'components/elements'
 import { Button, LinkButton } from 'components/form'
 import device from 'themes/device'
 import useHandleSignup from 'components/hooks/use-handle-signup'
 
+type DHeroProps = {
+    background?: string | JSX.Element
```

Comment:
Can we type it properly with `ImageDataLike | IGatsbyImageData` from gatsby-plugin-image?
Note: need to import just types like this: `import type { IGatsbyImageData, ImageDataLike } from 'gatsby-plugin-image'`

As far as I know `QueryImage` can get as background only these 2 types of properties

Reply:
okay